playback-sdk-android/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,34 @@
+<resources>
+
+    <!-- Following styles can be used to style the cast widgets -->
+    <style name="CustomCastExpandedController" parent="CastExpandedController"/>
+
+    <style name="CustomCastIntroOverlay" parent="CastIntroOverlay">
+        <item name="castButtonTextAppearance">@style/TextAppearance.CustomCastIntroOverlay.Button
+        </item>
+        <item name="castTitleTextAppearance">@style/TextAppearance.CustomCastIntroOverlay.Title
+        </item>
+    </style>
+
+    <style name="CustomCastMiniController" parent="CastMiniController">
+        <item name="castShowImageThumbnail">true</item>
+        <item name="castTitleTextAppearance">@style/TextAppearance.AppCompat.Subhead</item>
+        <item name="castSubtitleTextAppearance">@style/TextAppearance.AppCompat.Caption</item>
+    </style>
+
+    <style name="TextAppearance.CustomCastIntroOverlay.Button" parent="android:style/TextAppearance">
+        <item name="android:textColor">#FFFFFF</item>
+    </style>
```

Comment:
It would be great to create a color resource in the colors and put this one to the style item
`<item name="android:textColor">@color/white</item>`

or put the android resource color directly 
`<item name="android:textColor">@android:color/white</item>`

Reply:
Just copied and pasted the style of the bitmovin sample here: https://github.com/bitmovin/bitmovin-player-android-samples/blob/main/BasicCastingKotlin/src/main/res/values/styles.xml
But It's a good idea, so I'm going to change it.